src/sentry/static/sentry/app/views/organizationTeamsProjects.jsx

```diff
@@ -0,0 +1,162 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import {Box, Flex} from 'grid-emotion';
+import {Link} from 'react-router';
+import LazyLoad from 'react-lazy-load';
+
+import BarChart from '../components/barChart';
+import Button from '../components/buttons/button';
+import {Client} from '../api';
+import {loadStats} from '../actionCreators/projects';
+import Panel from './settings/components/panel';
+import PanelBody from './settings/components/panelBody';
+import PanelHeader from './settings/components/panelHeader';
+import PanelItem from './settings/components/panelItem';
+import ProjectLabel from '../components/projectLabel';
+import SettingsPageHeader from './settings/components/settingsPageHeader';
+import {t, tct} from '../locale';
+import withProjects from '../utils/withProjects';
+
+class OrganizationTeamsProjectsView extends React.Component {
+  static propTypes = {
+    params: PropTypes.object,
+    projects: PropTypes.array,
+  };
+
+  componentWillMount() {
+    this.api = new Client();
+  }
+
+  componentDidMount() {
+    this.fetchStats();
+  }
+
+  componentWillUnmount() {
+    this.api.clear();
+  }
+
+  fetchStats() {
+    loadStats(this.api, {
+      orgId: this.props.params.orgId,
+      query: {
+        since: new Date().getTime() / 1000 - 3600 * 24,
+        stat: 'generated',
+        group: 'project',
+      },
+    });
+  }
+
+  urlPrefix() {
+    let {orgId} = this.props.params;
+    return `/organizations/${orgId}`;
+  }
+
+  renderNoProjects(teamSlug) {
+    return (
+      <tbody>
+        <tr>
+          <td>
+            <p className="project-list-empty">
+              {tct(
+                'There are no projects in this team. Get started by [link:creating your first project].',
+                {
+                  link: <a href={this.urlPrefix() + '/projects/new/?team=' + teamSlug} />,
+                }
+              )}
+            </p>
+          </td>
+        </tr>
+      </tbody>
+    );
+  }
+
+  renderProject = project => {
+    let {orgId} = this.props.params;
+    let chartData =
+      project.stats &&
+      project.stats.map(point => {
+        return {x: point[0], y: point[1]};
+      });
+
+    // TODO(jess): prob should make sure they actually can manage projects
+
+    return (
+      <PanelItem key={project.id} align="center">
+        <Flex flex="1" justify="space-between">
+          <Link to={`/${orgId}/${project.slug}/`}>
+            <ProjectLabel project={project} />
+          </Link>
+          <div className="project-chart">
+            {chartData && (
+              <LazyLoad>
+                <BarChart points={chartData} label="events" />
+              </LazyLoad>
+            )}
+          </div>
+        </Flex>
+        <Box ml={2}>
```

Comment:
per your comment let's just kill this box when they dont have ``project:write`` permission

Reply:
👍 yeah i just wanted to get this up so you/chris could make sure it looks good. will add that now.